src/wmtk/attribute/Accessor.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "TupleAccessor.hpp"
+
+#include <Eigen/Dense>
+
+namespace wmtk::attribute {
+
+template <typename T>
+class Accessor: public TupleAccessor<T> {
```

Comment:
`Accessor` seems to be redundant. Maybe just replace it by `TupleAccessor`?

Reply:
i'm a bit torn on this - i like having the explicit semantic that a TupleAccessor is an Accessor. I guess i'll just replace TupleAccessor with Accessor (i don't want users to think about the Tuple aspect too much)